(be): build warnings

### Description

dotnet watch ⌚ D:\git\ivy\Ivy-Framework\Ivy\Widgets\VideoPlayer.cs(24,17): warning CS1573: Parameter 'poster' has no matching param tag in the XML comment for 'VideoPlayer.VideoPlayer(string?, bool, bool, bool, bool, string?)' (but other parameters do) [D:\git\ivy\Ivy-Framework\Ivy\Ivy.csproj]

### Steps to Reproduce

_No response_

### Expected Behavior

_No response_

### Actual Behavior

_No response_

### Environment

_No response_

### Code Samples

```csharp

```

### Screenshots

_No response_

### Additional Context

_No response_

--- a/Ivy/Widgets/VideoPlayer.cs
+++ b/Ivy/Widgets/VideoPlayer.cs
@@ -15,6 +15,7 @@ public record VideoPlayer : WidgetBase<VideoPlayer>
     /// <param name="controls">Whether playback controls should be displayed.</param>
     /// <param name="muted">Whether the video should be muted by default.</param>
     /// <param name="loop">Whether the video should loop continuously.</param>
+    /// <param name="poster">URL of an image to display before the video starts or while loading.</param>
     public VideoPlayer(
         string? source = null,
         bool autoplay = false,

